app/build.gradle.kts

```diff
@@ -155,6 +156,7 @@ android {
       isDebuggable = false
       isMinifyEnabled = runProguard.toBoolean()
       isShrinkResources = runProguard.toBoolean()
+      signingConfig = if(properties.get("maestroTest") == "true") getByName("debug").signingConfig else null
```

Comment:
Any reason we aren't using `maestroTest.toBoolean()` directly here?

Reply:
some changes got pushed accidentally, which pushing my first PR via Graphite. Will update it.